src/components/Category/CategoryContents/style.ts

```diff
@@ -4,10 +4,12 @@ export const St = {
   FlexContainer: styled.div`
     display: flex;
     justify-content: center;
+    width: 100%;
```

Comment:
```suggestion
  FlexContainer: styled.div`
    display: flex;
    justify-content: center;
```

div나 section은 width: 100% 가 디폴트라서 지워줘도 좋을 듯 해요!~